setup.py

```diff
@@ -16,35 +16,39 @@
 readme = open('README.rst').read()
 history = open('CHANGES.rst').read()
 
-sphinx_require = 'Sphinx>=1.5.1'
+sphinx_require = 'Sphinx>=3'
 
 tests_require = [
     'pytest-mock>=1.6.0',
-    'pytest-invenio>=1.3.3,<2.0.0',
-    'invenio-accounts>=1.3.0,<2.0.0',
-    'invenio-app>=1.3.0,<2.0.0',
+    'pytest-invenio>=1.4.1',
+    'invenio-accounts>=1.4.3',
+    'invenio-app>=1.3.0',
+    'SQLAlchemy-Continuum>=1.3.6',
+    'psycopg2-binary>=2.8.6',
     sphinx_require,
 ]
 
 # Should follow inveniosoftware/invenio versions
-invenio_db_version = '>=1.0.5,<2.0.0'
-invenio_search_version = '>=1.4.0,<2.0.0'
+invenio_search_version = '>=1.4.0,<1.5.0'
+invenio_db_version = '>=1.0.5,<1.1.0'
 
 extras_require = {
+    # Invenio-Search
     'elasticsearch6': [
-        'invenio-search[elasticsearch6]>={}'.format(invenio_search_version),
+        f'invenio-search[elasticsearch6]{invenio_search_version}'
```

Comment:
comment: nothing against this change, but is not needed. It is (IMO) a styling preference. The main issue is that by changing it if in the future there is a need to know when this got introduced it will point to you (and this commit) when in reality it comed form much earlier in time. Making finding the reason much harder. In this specific case, I is ok (that's why is a _comment_) because is so trivial that I doubt there are problems. But I mention just to keep it in mind for the future.

Reply:
this change I did is only that Github actions cannot find the versions see errors from github-actions:
```console 
Run pip install -r .min-3.6-requirements.txt
ERROR: Could not find a version that satisfies the requirement invenio-search[elasticsearch6]==>=1.4.0
ERROR: No matching distribution found for invenio-search[elasticsearch6]==>=1.4.0
Error: Process completed with exit code 1.
```

Reply:
Yes, good catch - basically i don't see why it has ever worked when it previously have generated this ``>=>=`` inside the string.